map/Region.dhall

```diff
@@ -0,0 +1,175 @@
+let Region =
+      < af-south-1
+      | ap-east-1
+      | ap-northeast-1
+      | ap-northeast-2
+      | ap-northeast-3
+      | ap-south-1
+      | ap-southeast-1
+      | ap-southeast-2
+      | ca-central-1
+      | cn-north-1
+      | cn-northwest-1
+      | eu-central-1
+      | eu-north-1
+      | eu-south-1
+      | eu-west-1
+      | eu-west-2
+      | eu-west-3
+      | me-south-1
+      | sa-east-1
+      | us-gov-east-1
+      | us-gov-west-1
+      | us-east-1
+      | us-east-2
+      | us-west-1
+      | us-west-2
+      >
+
+let render =
+      { computer =
```

Comment:
Please add a render for the partition of the region:

`aws` for normal AWS regions
`aws-cn` for AWS China regions
`aws-us-gov` for AWS GovCloud (US) regions

See: https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html